libraries/rush-lib/src/cli/RushXCommandLine.ts

```diff
@@ -206,6 +218,7 @@ export class RushXCommandLine {
     if (unknownArgs.length > 0) {
       // Future TODO: Instead of just displaying usage info, we could display a
       // specific error about the unknown flag the user tried to pass to rushx.
+      console.log(colors.red(`Unknown arguments: ${unknownArgs.join(', ')}`));
```

Comment:
This was useful while testing, and I can't see a reason not to keep it.

Reply:
Arguments are usually joined using spaces, not `,`.

Reply:
fixed